src/components/list/ListHeader.js

```diff
@@ -53,23 +53,22 @@ const StickyBackgroundBlocker = styled.View({
 export default function ListHeader({ children, contextMenuOptions, isCoinListEdited, showDivider = true, title, totalValue }) {
   const deviceDimensions = useDimensions();
   const { colors, isDarkMode } = useTheme();
-  const isReadOnlyWallet = useIsReadOnlyWallet();
   const accountAddress = useAccountAddress();
   const { accountENS } = useAccountProfileInfo();
   const { initializeShowcaseIfNeeded } = useWebData();
 
   const handleShare = useCallback(() => {
-    if (!isReadOnlyWallet) {
+    if (!getIsReadOnlyWallet()) {
```

Comment:
nitpick, but could probably cache the read only value here and use it in both places in this function?